modules/jaeger-config-collector.adoc

```diff
@@ -0,0 +1,43 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="jaeger-config-collector_{context}"]
+= Jaeger Collector configuration options
+
+The Jaeger Collector is the component responsible for receiving the spans that were captured by the tracer and writing them to a persistent storage (Elasticsearch) when using the `production` strategy, or to AMQ Streams when using the `streaming` strategy.
+
+The collectors are stateless and thus many instances of Jaeger Collector can be run in parallel. Collectors require almost no configuration, except for the location of the Elasticsearch cluster.
+
+.Jaeger collector parameters
```

Comment:
I think in this table, if there are no default values, we should remove that column.

Reply:
Done.